pkgs/applications/science/logic/redprl/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit, mlton }:
+stdenv.mkDerivation {
+  name = "redprl-2016-09-22";
+  src = fetchgit {
+    url = "https://github.com/RedPRL/sml-redprl.git";
+    rev = "3215faf0d494f4ac14d6e10172329a161df192c4";
+    sha256 = "0pcq4q9xy34j7ziwbly4qxccpkcrl92r9y11bv6hdkbzwm1g2a77";
+    fetchSubmodules = true;
+  };
+  buildInputs = [ mlton ];
+  builder = builtins.toFile "builder.sh" ''
+    source $stdenv/setup
+    mkdir -p $out/bin
+    cp -r $src/* .
+    chmod -R +w src
+    ./script/mlton.sh
+    mv ./bin/redprl $out/bin
+  '';
+  meta = {
```

Comment:
Being listed as a maintainer currently just means you care about the package and don't mind being summoned if there's an issue. Adding new orphans doesn't make too much sense. I mean, if you're not interested in keeping it up-to-date or keep it working, why add it?